pkg/restore/restore.go

```diff
@@ -848,6 +848,7 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 					addToResult(&warnings, namespace, e)
 				}
 			}
+			ctx.log.Infof("Unable to restore %s: %v because it already exists and is identical to the backed up copy", obj.GroupVersionKind().Kind, name)
```

Comment:
I think this needs to go into an `else` block, right? Otherwise, there are a couple of cases from the "not equal" if block above that will fall through and hit this, which will be incorrect.  If I'm reading it correctly the continue should remain outside the `else` since we want to continue for any case where we hit an AlreadyExists error.

Reply:
Ah, hmmm, good point.

Reply:
Which of 3 `if` statements would it be more pertinent to attach the `else` clause to? You think it might be worth refactoring by joining the 3 ifseses by `else if` and then finally `else`?